aten/src/ATen/native/mkldnn/TensorShape.cpp

```diff
@@ -40,6 +44,12 @@ Tensor mkldnn_reshape(const Tensor& self, IntArrayRef size) {
   return new_with_itensor_mkldnn(std::move(y), self.options());
 }
 
+Tensor mkldnn_clone(const Tensor& self) {
+  ideep::tensor& src = itensor_from_mkldnn(self);
+  ideep::tensor dst{src};
```

Comment:
this does make a copy, right? (it's supposed to)

Reply:
yes it's a copy ctor https://bddppq.github.io/codebrowser/pytorch/pytorch/third_party/ideep/include/ideep/tensor.hpp.html#1198